src/main/java/org/jahia/modules/sam/healthcheck/probes/SupportedStackJSModulesProbe.java

```diff
@@ -0,0 +1,81 @@
+package org.jahia.modules.sam.healthcheck.probes;
+
+import org.graalvm.polyglot.Context;
+import org.jahia.bin.Jahia;
+import org.jahia.commons.Version;
+import org.jahia.modules.sam.Probe;
+import org.jahia.modules.sam.ProbeSeverity;
+import org.jahia.modules.sam.ProbeStatus;
+import org.jahia.osgi.BundleUtils;
+import org.osgi.framework.Bundle;
+import org.osgi.service.component.annotations.Component;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+
+@Component(service = Probe.class, immediate = true)
+public class SupportedStackJSModulesProbe implements Probe {
+
+    @Override
+    public ProbeStatus getStatus() {
+        Bundle npmModulesEngineBundle = BundleUtils.getBundleBySymbolicName("npm-modules-engine", null);
+        boolean isNpmModulesEngineStarted = npmModulesEngineBundle != null && npmModulesEngineBundle.getState() == Bundle.ACTIVE;
+
+        String vmVendor = System.getProperty("java.vm.vendor");
+        Version jdkVersion = new Version(System.getProperty("java.version", "Unknown"));
+
+        // This probe is only relevant for Jahia 8.2.0.0+ in which npm-modules-engine is available. 
+        // Not testing Jahia version since it is not to be backported to older versions of SAM.
```

Comment:
yes, but actually since the main branch SAM can only be installed on Jahia 8.2x, I'll clean up the code and remove everything that could be faced with Jahia 81x. If we made the probe available in Jahia 81x, this will not be a backport, but that's probably cleaner this way.